CONTRIBUTING.md

```diff
@@ -275,6 +275,16 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### How to adopt pull requests
+
+Occasionally authors of pull requests get busy or become unresponsive. This is a natural part of any open source project. To avoid blocking these efforts, these pull requests may be *adopted*, where another contributor creates a new pull request with the same content. If there is an old Pull request without the label that is without updates, comment to the organization whether it is appropriate to add the *[S-Adopt-Me](https://github.com/bevyengine/bevy/labels/S-Adopt-Me)* label, to indicate that it can be *adopted*. Otherwise, should leave a comment on the PR asking the author if they plan on returning, if the author gives permission or simply doesn't respond after few days, then it can be adopted. This may sometimes even skip the labeling since at that point the PR has been adopted.
```

Comment:
> Occasionally authors of pull requests get busy or become unresponsive. 

This could make it seems like it's all on the authors shoulders... Sometimes it's also on the project side that we fail to reply in a timely manner

Reply:
Changes have been done @mockersf.